docs/source/conf.py

```diff
@@ -131,7 +131,7 @@ def setup(app):
 rediraffe_redirects = {
     "typedoc/index": "_static/typedoc/index",
     "try/lab/index": "_static/lab/index",
-    "try/retro/index": "_static/retro/tree/index",
+    "try/notebook/index": "_static/tree/index",
```

Comment:
Should this be `tree` instead, so it matches the value set above?

Reply:
Indeed!